### PR TITLE
dev analytics

### PR DESCRIPTION
**Pull request by peppasnet**

### CODE REVIEW

File: applications_dev/analytics/superset.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
```
Comment (bialask-daiteap) [inline applications_dev/analytics/superset.yaml:3]: We won't use dev any more. Please apply changes to dev2

**Review by bialask-daiteap — CHANGES_REQUESTED** [review]

**Review by bialask-daiteap — APPROVED** [review]

File: ionos_dev/analytics/superset/templates/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: superset
+            port:
+              number: 8088
+  tls:
+  - hosts:
+    - analytics.dome-marketplace-dev.org
```
Comment (outscale-hmi) [inline ionos_dev/analytics/superset/templates/ingress.yaml]: idem here

File: ionos_dev/analytics/superset/templates/ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  name: superset-ingress
+  namespace: analytics
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-dev-issuer
+    nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
+spec:
+  ingressClassName: nginx
+  rules:
+  - host: analytics.dome-marketplace-dev.org
```
Comment (outscale-hmi) [inline ionos_dev/analytics/superset/templates/ingress.yaml]: Should it be analytics.dome-marketplace-dev2.org

File: ionos_dev/analytics/superset/templates/ingress.yaml (hunk 2 above)
Comment (peppasnet) [reply ionos_dev/analytics/superset/templates/ingress.yaml]: Thats a deleted branch/PR. 
The valid PR is already merged with main. There are no analytics folder on "ionos_dev".
I'll check if there is a related/same problem on main branch on dev2 analytics folder and I'll create a new branch/PR in that case. Thanks.

File: ionos_dev/analytics/superset/templates/ingress.yaml (hunk 3 above)
Comment (peppasnet) [reply ionos_dev/analytics/superset/templates/ingress.yaml]: Thats a deleted branch/PR. 
The valid PR is already merged with main. There are no analytics folder on "ionos_dev".
I'll check if there is a related/same problem on main branch on dev2 analytics folder and I'll create a new branch/PR in that case. Thanks.

**Pull request merged**